runtime/browser/xwalk_browser_main_parts.cc

```diff
@@ -27,6 +27,9 @@
 #include "xwalk/application/browser/application_system.h"
 #include "xwalk/extensions/browser/xwalk_extension_service.h"
 #include "xwalk/extensions/common/xwalk_extension_switches.h"
+#if defined(USE_GTK_UI)
```

Comment:
Since all the GTK dependencies are summarized in 'libgtk2', including File Picker and some Input Methods supporting, and I believe we will revert this patch eventually, so, can we just use the name by now? Or any suggestion is fine for me.